packages/makeup-floating-label/src/index.js

```diff
@@ -44,6 +95,10 @@ function isFocused(textboxEl) {
     return document.activeElement === textboxEl;
 }
 
+function isSelect(textboxEl) {
+    return textboxEl.tagName === 'SELECT';
```

Comment:
This is a little odd. The name of `textboxEl` in some places might confuse us when we come back to this code later. Maybe `controlEl`or `formControlEl`?

Reply:
Or maybe just different functions for each, e.g. `mutateTextboxPlaceholder` and `mutateSelectPlaceholder`, `setTextboxPlaceholder` and `setSelectPlaceholder`, etc.

Reply:
I will go with `formControlEl`.
I think the logic is nice in these since they share 90% of the same logic. That way, only the setters and getters are different.